app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -192,22 +193,63 @@ class SamplesView extends React.Component {
         ? mapSidebarSelectedSampleIds
         : selectedSampleIds;
 
-    const log = () =>
-      logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
-        selectedSampleIds: targetSampleIds.length,
+    const heatmapOptions = [
+      { text: "Taxon Heatmap", value: "/visualizations/heatmap" },
+    ];
+
+    if (this.props.admin) {
+      heatmapOptions.push({
+        text: "AMR Heatmap",
+        value: "/amr_heatmap",
       });
-    return targetSampleIds.size < 2 ? (
-      <HeatmapIcon className={cx(cs.icon, cs.disabled, cs.heatmap)} />
-    ) : (
-      <a
-        onClick={log}
-        href={`/visualizations/heatmap?sampleIds=${Array.from(
-          targetSampleIds
-        )}`}
-      >
-        <HeatmapIcon className={cx(cs.icon, cs.heatmap)} />
-      </a>
-    );
+
+      return targetSampleIds.size < 2 ? (
+        <HeatmapIcon className={cx(cs.icon, cs.disabled, cs.heatmap)} />
+      ) : (
+        <BareDropdown
+          hideArrow
+          className={cx(cs.icon, cs.heatmapDropdown)}
+          closeOnClick={true}
+          items={heatmapOptions.map(option => {
+            const params = Array.from(targetSampleIds).join("&sampleIds[]=");
```

Comment:
Nit: Use `getURLParamString` from `helpers/url`. Good practice to use existing code when possible, though I realize it's very little code in this case.